README.md

```diff
@@ -36,6 +36,37 @@ Alternatively, copy the code for the individual macro directly from the .BAS fil
 12. After the macro is saved then it can be mapped to a user tool, key, or key chord as usual (optional).
```

Comment:
After the macro is saved then it can be mapped to a user tool add added to the toolbar, or mapped to a keyboard shortcut as usual (optional).
The OCLC terminology is Keymap and Shortcut Keys